Magento_Theme/templates/html/rwahsnav.phtml

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Copyright © 2015 Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+?>
+<div class="rwahsglobalnav">
+    <ul class="nav nav-tabs">
+        <li role="presentation"><a href="https://uat-www.histwest.org.au">Website</a></li>
```

Comment:
These should be production links, not UAT links.  There's a risk that we'll forget to change it, and I'd rather have it incorrectly point to production from UAT, than incorrectly point to UAT from production.